evic/device.py

```diff
@@ -161,6 +167,34 @@ def read_dataflash(self):
 
         return (dataflash, checksum)
 
+    def fmc_read(self, start, length):
```

Comment:
I'd add a small note to the docstring, mentioning `fmc_read` and `read_screen` are specific to your FW.